src/ManifoldsBase.jl

```diff
@@ -480,8 +556,29 @@ function is_point(M::AbstractManifold, p, throw_error = false; kwargs...)
     return throw_error ? throw(mpe) : false
 end
 
+function is_point(M::AbstractManifold, p, error::Symbol; kwargs...)
+    (error === :error) && return is_point(M, p, true; kwargs...)
+    mps = check_size(M, p)
+    if mps !== nothing
+        s = "$(typeof(mps)) with $(mps.val)\n$(mps.msg)"
+        (error === :info) && @info s
+        (error === :warning) && @warn s
+        return false
+    end
+    mpe = check_point(M, p; kwargs...)
+    if mpe !== nothing
+        s = "$(typeof(mpe)) with $(mpe.val)\n$(mpe.msg)"
+        (error === :info) && @info s
+        (error === :warning) && @warn s
+        return false
+    end
+    return true
+end
+
+
 """
     is_vector(M::AbstractManifold, p, X, throw_error = false, check_base_point=true; kwargs...)
+    is_vector(M::AbstractManifold, p, X, error::Symbol, check_base_point=true; kwargs...)
```

Comment:
```suggestion
    is_vector(M::AbstractManifold, p, X, error::Symbol, check_base_point::Bool=true; kwargs...)
```
Since we are working on this part anyway, we could add `::Bool` here to be more precise.